src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/Unix/KeyAgreeRecipientInfoPalOpenSsl.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Security.Cryptography;
+using System.Security.Cryptography.Pkcs;
+
+namespace Internal.Cryptography.Pal.OpenSsl
+{
+    internal sealed class KeyAgreeRecipientInfoPalOpenSsl : KeyAgreeRecipientInfoPal
+    {
+        public override DateTime Date
+        {
+            get
+            {
+                throw new NotImplementedException();
```

Comment:
With the OpenSSL API, I'm going to have to think what's the best way to address these, but for now I plan to address these at the end, after getting Key Transport working, so for now, I feel this is the appropriate thing to throw.